AWSAuthSDK/AWSAuthSDK.xcodeproj/project.pbxproj

```diff
@@ -199,6 +199,21 @@
 		B9AC6319234F00FA00A713A5 /* AWSAuthUIHelper.m in Sources */ = {isa = PBXBuildFile; fileRef = B9AC6317234F00F900A713A5 /* AWSAuthUIHelper.m */; };
 		B9AC631A234F00FA00A713A5 /* AWSAuthUIHelper.h in Headers */ = {isa = PBXBuildFile; fileRef = B9AC6318234F00FA00A713A5 /* AWSAuthUIHelper.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		B9E2F6CD231EEE63009EAA48 /* AWSMobileClientInitializerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = B9E2F6CC231EEE63009EAA48 /* AWSMobileClientInitializerTests.swift */; };
+		D8FAC273251D1A87005F9FC6 /* FBSDKCoreKitImport.h in Headers */ = {isa = PBXBuildFile; fileRef = D8FAC202251D1A87005F9FC6 /* FBSDKCoreKitImport.h */; };
```

Comment:
What are these changes? Below I only see some file modified and no new files.

Reply:
When I finished replacing all header files and opened project file, I found out that not every header files are in the project file. So I dragged them one by one. Some files in v7 are the same with v6, I believe that's the reason you are not seeing new files.

Reply:
In this case, it looks like the opposite: There were files in the directory tree that were not referenced in the project file. This commit fixes that.